server/index.php

```diff
@@ -23,6 +23,7 @@
 $app->delete('/users', 'UserManager:delete');
 $app->get('/validate/email/{token}', 'EmailValidator:validate')->setName('validateEmail');
 $app->get('/status', 'Status:status');
+$app->get('/instances', 'InstanceManager:getInstances');
```

Comment:
I'm unsure if we want to print all instances of a lookup server always. I guess on a public lookup server we don't want this, right? cc @rullzer

Reply:
can I use the settings _'GLOBAL_SCALE' => false,_ to filter/enable the endpoint ?

Reply:
Might be an option. But then also make sure to not expose the lookup server to anything in the internet, but only locally in the network of the nodes.